src/sources/forus-webshop/js/angular-1/components/HomeComponent.js

```diff
@@ -28,12 +28,15 @@ let HomeComponent = function(
         return ModalService.open('modalOpenInMe', {});
     };
 
-    if(AuthService.hasCredentials()) {
-        VoucherService.list().then(res => {
-            $ctrl.vouchers = res.data.data; });
+    $ctrl.$onInit = function() {
```

Comment:
why do we change this on this branch? is this intended?

Reply:
just some code refactoring, it wont really affect anything